build_posix/aclocal/strict.m4

```diff
@@ -138,6 +138,11 @@ AC_DEFUN([AM_CLANG_WARNINGS], [
 	# conditional body.
 	w="$w -Wno-extra-semi-stmt"
 
+	# TODO: Figure out whether we want to disable these or change the code.
+	w="$w -Wno-implicit-int-float-conversion"
+	w="$w -Wno-implicit-fallthrough"
+	w="$w -Wno-maybe-uninitialized"
+
```

Comment:
Can you share the context about these new warning flags?

Reply:
Sure, I will also put a comment on WT-8052,

1. `implicit-int-float-conversion` : The original error that was being generated does not prefer the conversion of uint64_t to double due to loss of precision. 
2. `implicit-fallthrough` : The compiler generates the error as it thinks that the programmer forgot to put a break statement in a switch statement. However, I am sure that missing break is intentional in the code.
3.  `maybe-uninitialized`: The compiler generates these warnings as it thinks that a variable is uninitialized, however, it also generates those warnings if the variable was inititalized within a conditional statement. The later is the case with our code.